config/install/field.field.paragraph.social_link.field_paragraph_link.yml

```diff
@@ -12,11 +12,11 @@ entity_type: paragraph
 bundle: social_link
 label: Link
 description: ''
-required: false
+required: true
 translatable: true
 default_value: {  }
 default_value_callback: ''
 settings:
   link_type: 16
-  title: 1
+  title: 2
```

Comment:
Hey @conantran what is this value?

Reply:
Hey @anthony-malkoun, this value set to 2 means the link text is also required field, it was set to optional. This is needed per the ticket. See screenshot.
<img width="740" alt="SDPA-1617-link-text" src="https://user-images.githubusercontent.com/10159742/55843454-acd11780-5b7b-11e9-8680-e35bf47bc690.png">